File: frontend/src/types/InKindDonationItem.ts
export interface InKindDonationItem {
  itemId: number;
  donationId: number;
  itemName?: string;
  itemCategory?: string;
  quantity?: number;
  unitOfMeasure?: string;
  estimatedUnitValue?: number;
  intendedUse?: string;
  receivedCondition?: string;
}

File: backend/Beacon.API/Models/InKindDonationItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Beacon.API.Models;

public partial class InKindDonationItem
{
    [Key]
    public int ItemId { get; set; }

    public int DonationId { get; set; }

    public string? ItemName { get; set; }

    public string? ItemCategory { get; set; }

    public int? Quantity { get; set; }

    public string? UnitOfMeasure { get; set; }

    public decimal? EstimatedUnitValue { get; set; }

    public string? IntendedUse { get; set; }

    public string? ReceivedCondition { get; set; }

    public virtual Donation Donation { get; set; } = null!;
}
